SnapMD.ConnectedCare.ApiModels/PatientConsultationInfo.cs

```diff
@@ -24,6 +24,11 @@ public class PatientConsultationInfo
         public string AssignedDoctorFirstName { get; set; }
         public string AssignedDoctorLastName { get; set; }
 
+        /// <summary>
+        /// For patients who depend from they guardians this flag shows whether patient authorized by any of his guardians or not.
+        /// </summary>
+        public bool IsAuthorized { get; set; }
```

Comment:
@simplecode80 @Number1MSFanboy This should be handled differently. The property does not indicate that the dependent Patient is "authorized", but that the current user is "authorized" to handle the dependent's matters. Therefore it should be something like a "GuardianAuthorizationStatus" and incude a collection of user IDs for users allowed to manage the user's health matters.